tests/regression/lp-1805838/task.yaml

```diff
@@ -6,21 +6,41 @@ details: |
 prepare: |
     #shellcheck source=tests/lib/dirs.sh
     . "$TESTSLIB"/dirs.sh
-    mount --bind /bin/false "$LIBEXECDIR/snapd/snap-seccomp"
-    test -d "$SNAP_MOUNT_DIR/core" && mount --bind /bin/false "$SNAP_MOUNT_DIR/core/current/usr/lib/snapd/snap-seccomp"
-    test -d "$SNAP_MOUNT_DIR/snapd" && mount --bind /bin/false "$SNAP_MOUNT_DIR/snapd/current/usr/lib/snapd/snap-seccomp"
+    #shellcheck source=tests/lib/snaps.sh
+    . "$TESTSLIB"/snaps.sh
+    install_local network-consumer
+
+    version_info="$(sed -n '2 s/# //p' < /var/lib/snapd/seccomp/bpf/snap.network-consumer.network-consumer.src)"
```

Comment:
Would it not be easier to run real snap-seccomp and get the version info there?

Reply:
Uhh, yeah, probably.

Reply:
Ping about this?

Reply:
Please look at the commit log, specifically https://github.com/snapcore/snapd/pull/6485/commits/63a3fe43911a34bd484520c5e4e5b580e0bf8139 and the next one. I did the change and then reverted it back, because there's some extra assumptions made by snapd about which tool to use, which felt too complicated to replicate here.